packages/postgresql.rb

```diff
@@ -27,7 +27,7 @@ class Postgresql < Meson
   depends_on 'libxml2' # R
   depends_on 'libxslt' # R
   depends_on 'linux_pam' # R
-  depends_on 'llvm18_dev' => :build
+  depends_on 'llvm20_dev' => :build
```

Comment:
This could be `depends_on 'llvm_dev'` to abstract the version.

Reply:
Done.